google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/CloudDebugRunConfiguration.java

```diff
@@ -147,8 +146,7 @@ public void setProcessState(@Nullable CloudDebugProcessState processState) {
   @Nullable
   @Override
   public RunProfileState getState(
-      @NotNull Executor executor, @NotNull ExecutionEnvironment environment)
-      throws ExecutionException {
+      @NotNull Executor executor, @NotNull ExecutionEnvironment environment) {
```

Comment:
just curious - why did you get rid of the `throws`?

Reply:
This method does not throw any checked exceptions (probably it used to?).